documentation/userguide.md

```diff
@@ -410,6 +410,11 @@ off the proxy altogether and for using a different proxy.
 
 By default, the upload daemon will not validate SSL certificates. We strongly recommend turning the validation on by setting the `disableSslValidation` option in the config file to `false`. It is disabled by default to allow for an easier setup in environments that use self-signed certificates or custom certificate authorities.
 
+## Upload Interval
+
+By default, the upload daemon will upload new trace files every 5 minutes. To change this interval, set the `uploadIntervalInMinutes` option in the config file to the desired value.
```

Comment:
change to "will upload finished trace files"? As new trace files are locked and cannot be uploaded yet.